ontology/yaml/resources/SAFETY/entity_types/FD.yaml

```diff
@@ -19,4 +19,15 @@ FD_STD:
   implements:
   - FD
   uses:
-  - damper_status
\ No newline at end of file
+  - damper_status
+  
+ FD_SD:
+  description: "Basic smoke detector fire damper"
+  is_canonical: true
+  implements:
+  - FD
+  - SD
```

Comment:
You cannot implement this because it is not in the same namespace. You also have defined damper controls below, but omitted the `supply`. You intend to have a device with both `supply_air_damper_percentage_command` and `damper_percentage_command`? Please explain.

Reply:
Oh hehe. I saw in SAFETY\GENERALTYPES the abstract type "SD" as smoke detector, which is what I needed sir :) this is correct, i think, fire damper that's a smoke detector, it is nearest for the device we have.

Reply:
yes sir it is contained within but no data other than dampers are available


Motorized Smoke Fire Damper
```
Open/Close Feedback
Open/Close Control
```

Reply:
im ok with FD STD. we can rename later if necessary.